test/extensions/filters/http/ext_proc/unit_test_fuzz/ext_proc_unit_test_fuzz.cc

```diff
@@ -69,6 +69,14 @@ DEFINE_PROTO_FUZZER(
     return;
   }
 
+  // Limiting the max supported request body size to 128k.
+  if (input.request().has_proto_body()) {
```

Comment:
Is this needed? 128k is applicable to response not request.

Reply:
Thanks for reviewing.

There are an intermittent CI failure: https://dev.azure.com/cncf/envoy/_build/results?buildId=180586&view=logs&j=e969334a-0e55-5c18-ac96-8b546753391e&t=4122df52-7ea9-5745-8c7d-f6b6f2c73149&l=3788
 when run ext_proc_unit_test_fuzz with test case: 
https://github.com/envoyproxy/envoy/blob/aa195eb01d1211aa9e082f2d742d75395975edc0/test/extensions/filters/http/ext_proc/unit_test_fuzz/ext_proc_corpus/clusterfuzz-testcase-minimized-ext_proc_unit_test_fuzz-5375162511851520#L16
This test case has a input with request body ~1MB. Such large request body is causing this CI timeout. We can avoid such timeout by add "size = 'large'" in the BUILD file to give this fuzzer target more resource. Or limiting the input body size to a smaller number.  After talking to @adisuissa , it looks to me in fuzzer there is no need to test such a large body, thus added this limit.